packages/libs/react-ui/src/components/Notification/NotificationContainer.tsx

```diff
@@ -0,0 +1,64 @@
+import { SystemIcon } from '../Icons';
+
+import {
+  cardTitleClass,
+  colorVariants,
+  containerClass,
+  contentClass,
+  expandVariants,
+  iconContainerClass,
+  iconContainerExpandedClass,
+} from './Notification.css';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcon)[keyof typeof SystemIcon];
+  title?: string;
+  children?: React.ReactNode;
+  expanded?: boolean;
+  color?: keyof typeof colorVariants;
+  hasCloseButton?: boolean;
+  onClose?: () => void;
+}
+
+export const NotificationContainer: FC<INotificationProps> = ({
+  icon,
+  title,
+  children,
+  hasCloseButton = false,
+  color = 'primary',
+  expanded = false,
+  onClose,
+}) => {
+  const Icon = icon || SystemIcon.HelpCircle;
+
+  const classList = classNames(
+    containerClass,
+    colorVariants[color],
+    expandVariants[expanded ? 'true' : 'false'],
+  );
+
+  return (
+    <div className={classList}>
+      <span className={iconContainerClass}>
+        <Icon size={'md'} />
+      </span>
+
+      <div>
+        <p className={contentClass}>
+          {title !== undefined && <h4 className={cardTitleClass}>{title}</h4>}
+          <p>{children}</p>
+        </p>
+      </div>
+
+      <span
+        onClick={onClose}
+        className={expanded ? iconContainerExpandedClass : iconContainerClass}
+      >
+        {hasCloseButton && <SystemIcon.Close size={'md'} />}
+      </span>
```

Comment:
I'm confused by this. Should this be a button?

Reply:
It does make sense to make it a button, although it won't be aligned with the other icon, nor the title:
![notification-icon-button](https://github.com/kadena-community/kadena.js/assets/2262851/60d80f4c-37b8-477c-a4b1-ada0cae24900)

should we go with it?

Reply:
I actually mean it's alignment to the top and right ends of the card, as the `IconButton` component defines their own sizes/margins

Reply:
Ahh yes, in this case I also think it makes sense to give it custom styles rather than reusing the IconButton. The main point is that it needs to be a `button` element so for accessibility. Right now you can tab to the close button to close it because it just uses an onclick handler. It does look like the color is also off so that would be a good thing to fix as well

Reply:
I see!

Ended up re-using the same button component, `NotificationButton`, as the one used for the bottom actions, with the possibility to not include text. Looks alright and aligns properly 🥳